homeassistant/components/automation/blueprints/notify_leaving_zone.yaml

```diff
@@ -1,5 +1,5 @@
 blueprint:
-  name: Zone Notification
+  name: Leave Zone Notification
```

Comment:
Isn't the description enough?

Reply:
I thing not enough, in UI description `Send a notification to a device when a person leaves a specific zone.` it's too small to pay attention. For new or inattentive users, this will lead to misunderstandings and worsen the user experience, since users will think that the notification does not trigger when entering the zone.

Reply:
If you read the description, why would you think that it notifies when entering a zone?

Reply:
The key word is "if", as I say before, description too small to pay attention and title without clarification leads to misunderstanding. Also, since there is no blueprint to notify about entering in the zone, it seems that this blueprint performs both functions.

Reply:
I think you have to read the description to understand what the blueprint does. The name doesn't say anything about entering or leaving. I don't see why one would assume that the blueprint is about both entering and leaving a zone when just reading the blueprint name.

The purpose of the name is not to explain exactly what the blueprint does. The purpose is to identify the blueprint. It's the description that should explain what the blueprint does.